Rules/CommonScripts/ChatCommands/AnimalCommands.as

```diff
@@ -0,0 +1,59 @@
+#include "ChatCommand.as"
+
+class ChickensCommand : ChatCommand
+{
+	ChickensCommand()
+	{
+		super("chickens", "Spawn a flock of chickens");
+		AddAlias("chickenflock");
+	}
+
+	void Execute(string[] args, CPlayer@ player)
+	{
+		if (!isServer()) return;
```

Comment:
Not a huge fan of the verbosity and duplicated code all the blob-spawning commands come with. Maybe a common blob spawning class could be used, so that its children would only have to define the constructor and implement a `void SpawnBlobAt(Vec2f pos)` class?